packages/allure-js-commons/src/writers/utils.ts

```diff
@@ -1,38 +1,909 @@
-import { AttachmentOptions, ContentType } from "../model";
+/* eslint-disable max-lines */
+import { AttachmentOptions } from "../model";
+
+const extensionsByType: Record<string, string> = {
```

Comment:
Everything seems fine, but maybe we can add some test cases to make sure this part working fine. Even one positive scenario will be enough

Reply:
done